### PR TITLE
bumping version, modifying docs

### PR DESCRIPTION
**Pull request by ian-scale**

### CODE REVIEW

File: docs/guides/endpoint_creation.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ An example is provided below:
 ```
 model_name = "llama-2-7b.suffix.2023-07-18-12-00-00"
 response = Model.get(model_name)
-while response.status != "READY":
+while response.status.name != "READY":
+    print(response.status.name)
     time.sleep(60)
```
Comment (yunfeng-scale) [inline docs/guides/endpoint_creation.md:12]: wait for 10 seconds? with recent chagnes we should be able to deploy things faster

**Review by yunfeng-scale — APPROVED** [review]

File: docs/guides/endpoint_creation.md (hunk 1 above)
Comment (yixu34) [reply docs/guides/endpoint_creation.md:12]: Hm what changes?

**Review by yixu34 — APPROVED** [review]

File: docs/guides/endpoint_creation.md (hunk 1 above)
Comment (yunfeng-scale) [reply docs/guides/endpoint_creation.md:12]: > what changes

Ray's cold start changes

Comment (ian-scale) [reply docs/guides/endpoint_creation.md:12]: Good to merge this in now?

Comment (yunfeng-scale) [inline docs/guides/endpoint_creation.md:12]:
```suggestion
    time.sleep(10)
```

**Pull request closed without merging**